openlibrary/templates/books/edit/edition.html

```diff
@@ -239,11 +239,11 @@
         <div class="formElement">
             <div class="label">
                 <label for="edition-title_other">$_("Is it known by any other titles?")</label>
-                <span class="tip">$_("Perhaps in another language?")</span>
+                <span class="tip">$_("Do different titles appear on the cover, spine, or title page? The individual titles of collections or anthologies may be entered here as well as.")</span>
```

Comment:
This text is a bit long, but it also looks like there's an extra word at the end of `The individual titles of collections or anthologies may be entered here as well as.` ?  I'm not sure exactly what that means -- it sounds like a list could be entered, but I don't know if that'll work here.

Other than the latter part of this line, the PR changes and intent are good!

Reply:
I must have accidentally deleted some text. I will fix.